CHANGELOG.md

```diff
@@ -10,11 +10,11 @@ _None_
 
 ### New Features
 
-_None_
-
+* Support for a version.properties to manage app versioning - all existing paths remain intact and new paths are only used when a version.properties file is present
+* Add support for providing an `app:` parameter to most versioning-related actions to allow support for multiple apps hosted in a monorepo
 ### Bug Fixes
```

Comment:
nit: empty line between sections (I think my release script which processes the changelog supports if you omit it? but better be sure)
```suggestion

### Bug Fixes
```

Reply:
Added along with other Changelog changes!